app/assets/javascripts/3dbio_viewer/src/webapp/components/protvista/ProtvistaPdbValidation.tsx

```diff
@@ -5,24 +5,83 @@ import { ProtvistaPdb, ProtvistaPdbProps } from "./ProtvistaPdb";
 declare global {
     const d3: typeof d3Module;
 }
+const data = [
+    {
+        name: "foo",
+        units: 32,
+    },
+    {
+        name: "bar",
+        units: 67,
+    },
+    {
+        name: "baz",
+        units: 81,
+    },
+    {
+        name: "hoge",
+        units: 38,
+    },
+    {
+        name: "plyo",
+        units: 28,
+    },
+    {
+        name: "hogera",
+        units: 59,
+    },
+];
+
+const dimensions = {
+    width: 450,
+    height: 300,
+};
 
 export const ProtvistaPdbValidation: React.FC<ProtvistaPdbProps> = React.memo(props => {
     console.log("tracks", props.pdb.tracks);
```

Comment:
I left it just as hint on where to find track info, but won't be needed for now. Let's remove it for the PR.